src/transformers/integrations/fbgemm_fp8.py

```diff
@@ -60,6 +60,7 @@ def forward(self, x):
         output = output + self.bias if self.bias is not None else output
         # Hacky for now, we have the output to the device of x
         output = output.to(x.device)
+        output = output.reshape((*x.shape[:-1], -1))
```

Comment:
Can we just store the original shape before and reshape it here ? it will be easier to go through the code.

Reply:
sounds reasonable, will do!